src/value.h

```diff
@@ -186,18 +184,11 @@ static inline RSValue *RS_ConstStringVal(const char *s, size_t n) {
 /* Wrap a redis string value */
 RSValue *RS_RedisStringVal(RedisModuleString *str);
 
-/**
- *  Create a new value object which increments and owns a reference to the string
- */
-RSValue *RS_OwnRedisStringVal(RedisModuleString *str);
```

Comment:
I can see this being used in the future. If this is dead code, why not add a test for it?